platform/engineering/design-docs/flag-for-manual-review.md

```diff
@@ -0,0 +1,113 @@
+# Flag for manual review process
+
+**Author(s):** Brooks Johnson (brooks@adhoc.team)
+**Last Updated:** February 20, 2020  
+**Status:** **Draft** | In Review | Approved  
+**Approvers:** _Rian Fowler_ \[ \], _Andrew Gunsch_ \[ \], ...  
+
+
+
+## Overview
+
+### Objective
+
+The goal is to improve our review process which by adding automated processes which will alert a review team when a manual review is required.
+These changes are meant to work alongside the Codeowners changes which will give each app team primary responsibility for reviewing their code.
+
+Non-goals: Creating automation which will absolve VFS teams of their review responsibility.
```

Comment:
```suggestion
```

I don't think this strikes the right tone. We want to automate the easy stuff so VFS teams can focus their reviews on the harder problems. For the things the platform must enforce but can't automate, we're using this trigger.

Reply:
That's fair - I'll remove it.